tutorials/03_multiround_inference.ipynb

```diff
@@ -62,28 +62,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<torch._C.Generator at 0x7f094007e490>"
-      ]
-     },
-     "execution_count": 8,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "import torch\n",
     "\n",
     "from sbi.inference import SNPE, prepare_for_sbi, simulate_for_sbi\n",
     "from sbi.utils.get_nn_models import posterior_nn\n",
     "from sbi import utils as utils\n",
     "from sbi import analysis as analysis\n",
-    "torch.manual_seed(0)"
+    "_ = torch.manual_seed(0)"
```

Comment:
why not just `torch.manual_seed(0)`?

Reply:
`torch.manual_seed(0)` returns `<torch._C.Generator object at 0x7f10e43d1a90>` which I found a bit ugly in the notebook ;)

Reply:
I see, makes sense then. Alternatively, one could add a `;` at the end of the line, I think.